scripts/similarity_trainer.py

```diff
@@ -48,7 +49,3 @@ def main(args):
         )
 
     model.save()
-
-
-if __name__ == "__main__":
-    main(parse_args(sys.argv[1:]))
```

Comment:
Instead of changing this, you can just add the script to "scripts" in setup.py instead of "console_scripts" (we'll do it for all scripts anyway soon https://github.com/mozilla/bugbug/issues/831).